docs/user-event/install.mdx

```diff
@@ -26,6 +26,16 @@ yarn add --dev @testing-library/user-event
 </TabItem>
 </Tabs>
 
+Now simply import it in your tests:
+
+```js
+import userEvent from '@testing-library/user-event'
+
+// or
+
+const {default: userEvent} = require('@testing-library/user-event')
```

Comment:
Shouldn't this be automatically happen when you write `import userEvent from '@testing-library/user-event'` and let it transpile by Babel?
This seems like a step backwards regarding progress of ES6 modules in the ecosystem so I would encourage `user-event` to get this right. You probably want to ship ES6 modules and CommonJS in `module` and `main` entrypoints respectively.

Reply:
> Shouldn't this be automatically happen when you write `import userEvent from '@testing-library/user-event'` and let it transpile by Babel?

Yes, but not everyone uses Babel. [Testing Library Recorder Extension](https://github.com/testing-library/testing-library-recorder-extension) needs to use CJS modules for now because it can't introduce configuration to enable Babel or Jest's experimental ES modules (which in my experience do not work well with that project's dependencies). I edited the documentation to fix this previously, but it wasn't included in the new docs for version 14.

> You probably want to ship ES6 modules and CommonJS in `module` and `main` entrypoints respectively.

user-event does do this. Are you suggesting this is a build configuration issue with user-event?